tests/test_metric_snapshot_df.py

```diff
@@ -72,6 +72,17 @@ def test_init_single_metric(self):
             "incorrect dataframe shape when initialized with single json list",
         )
 
+    def test_init_multiple_metrics(self):
+        """Ensures metric values provided as strings are properly cast to a numeric value (in this case, a float)."""
+        raw_data = [
+            {"metric": {"fake": "data",}, "value": [1627485628.789, "26.82068965517243"],},
+            {"metric": {"fake": "data",}, "value": [1627485628.789, "26.82068965517243"],},
+        ]
+
+        test_df = MetricSnapshotDataFrame(data=raw_data)
+
+        self.assertTrue(isinstance(test_df["value"][0], float))
```

Comment:
Could you add similar tests for the Metric and MetricSnapshotDataFrame classes as well?

Reply:
You bet 🙂 Will get these taken care of tomorrow.

Reply:
How's that look to you? Apologies for the delay

Reply:
Looks great, thanks! :smile: 
> similar tests for the Metric and MetricSnapshotDataFrame

Also, I just realized I meant Metric and _MetricRangeDataFrame_ here in this comment. So just a final nit, could you pls add tests for that too?